VideoLocker/src/main/java/org/edx/mobile/view/MyRecentVideosFragment.java

```diff
@@ -225,39 +224,31 @@ private void addToRecentAdapter(final View view) {
         }
         logger.debug("reloading adapter...");
         final String selectedId = adapter.getVideoId();
-
-
-        if (getRecentDownloadedVideosTask != null) {
```

Comment:
It seems that this was ensuring that if there was already an instance of this task in progress, then it would be cancelled before restarting another instance of it, in order to avoid race conditions.